src/Microsoft.Health.Dicom.Core/Features/CustomTag/CustomTagEntry.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+namespace Microsoft.Health.Dicom.Core.Features.CustomTag
+{
+    public class CustomTagEntry
+    {
+        public long TagId { get; set; }
+
+        public string Path { get; set; }
+
+        public string VR { get; set; }
+
+        public CustomTagLevel Level { get; set; }
+
+        public long Version { get; set; }
```

Comment:
What is the version for? Can you add comments on class and properties

Reply:
updated

Reply:
Thanks. How is it used here?

Reply:
Also can you give me an example of an allowed update and the effect on the re-indexing

Reply:
Version match to SQL TIMESTAMP type in database. Each CustomTag row in database has a version, every time it got changed, a new version is assigned by system automatically.
TIMESTAMP is a global bigint which keep increasing and never come back. The max version in CustomTag table is the version of while CustomTag table.


I'm thinking if create another table (CustomTagVersionTable) , and put the version into that is a more clear idea. We only need version of entire CustomTag table instead of each row.  The CustomTagVersionTable will look like:

CREATE TABLE dbo.CustomTagVersion (
    Version               TIMESTAMP               NOT NULL,
   -- Audit column
   LastStatusUpdatedDate                DATETIME2(7) NOT NULL,
)